eng/Version.Details.xml

```diff
@@ -153,9 +153,9 @@
       <Uri>https://github.com/dotnet/sourcelink</Uri>
       <Sha>4b584dbc392bb1aad49c2eb1ab84d8b489b6dccc</Sha>
     </Dependency>
-    <Dependency Name="XliffTasks" Version="1.0.0-beta.21265.1" CoherentParentDependency="Microsoft.DotNet.Arcade.Sdk">
+    <Dependency Name="XliffTasks" Version="1.0.0-beta.21262.1" CoherentParentDependency="Microsoft.DotNet.Arcade.Sdk">
```

Comment:
@sfoslund @dsplaisted Is this downgrade expected?

Reply:
Let me check with core-eng people who might know context for this @v-wuzhai.